cla_public/templates/base.html

```diff
@@ -129,5 +129,5 @@
   <li><a href="{{ url_for('base.online_safety') }}">{{ _('Staying safe online') }}</a></li>
   <li><a href="{{ url_for('base.privacy') }}">{{ _('Privacy Policy') }}</a></li>
   <li><a href="{{ url_for('base.cookies') }}">{{ _('Cookies') }}</a></li>
-  <li><a href="{{ url_for('base.feedback') }}">{{ _('Feedback') }}</a></li>
+  <li><a href="https://checklegalaid.service.gov.uk/reasons-for-contacting">{{ _('Contact') }}</a></li>
```

Comment:
:raising_hand_man: It's preferable to use `url_for` than to use a full domain URL as we can configure the former to work on other environments (currently all environments would use production for this link).

Reply:
Thanks for the review and comments @sldblog 

What would be the correct way of writing this? 

Can you make a suggestion like the last comment. That was really helpful way of committing.

Reply:
It sounds like we want to use `url_for('base.reasons_for_contacting')` to return `/reasons-for-contacting` everywhere as this template already exists:

https://github.com/ministryofjustice/cla_public/blob/bafcec056500df246bb8ba7df71c3bece3f18f64/cla_public/templates/reasons-for-contacting.html

@exonian and @said-moj can tell if I'm talking sense or not :joy:

Reply:
Makes sense! Will make the change.